dev/run-tests.py

```diff
@@ -815,11 +765,10 @@ def main():
 
     modules_with_python_tests = [m for m in test_modules if m.python_test_goals]
     if modules_with_python_tests:
-        # We only run PySpark tests with coverage report in one specific job with
-        # Spark master with SBT in Jenkins.
-        is_sbt_master_job = "SPARK_MASTER_SBT_HADOOP_2_7" in os.environ
         run_python_tests(
-            modules_with_python_tests, opts.parallelism, with_coverage=is_sbt_master_job)
+            modules_with_python_tests,
+            opts.parallelism,
+            os.environ.get("PYSPARK_CODECOV", "false") == "true")
```

Comment:
nit: it's better to stay the explict args name for python default value argument, this is a style prefer for me, just feel free to change or not. : )